xml/System.Text.RegularExpressions/GroupCollection.xml

```diff
@@ -103,9 +103,9 @@
         <Parameter Name="key" Type="System.String" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="key">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="key">A string with the name of the captured group to locate.</param>
+        <summary>Determines whether the group collection contains a captured group identified by the specified name.</summary>
+        <returns><see langword="true"/> if the groupcollection contains a captured group identified by the specified key; <see langword="false" /> otherwise.</returns>
```

Comment:
is it ok for groupcollection to be lowercase like this?

Reply:
Nope, my mistake. Fixing it:

```suggestion
        <returns><see langword="true"/> if the group collection contains a captured group identified by the specified key; <see langword="false" /> otherwise.</returns>
```